lib/app/features/search/views/components/advanced_search_channels/advanced_search_channels.dart

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/components/separated/separator.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/features/search/views/components/advanced_search_groups/advanced_search_group_list_item.dart';
+import 'package:ion/app/features/search/views/components/search_sub_header/search_sub_header.dart';
+import 'package:ion/generated/assets.gen.dart';
+
+class AdvancedSearchChannels extends HookConsumerWidget {
+  const AdvancedSearchChannels({required this.query, super.key});
+
+  final String query;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    useAutomaticKeepAlive();
+
+    return SingleChildScrollView(
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Padding(
+            padding: EdgeInsets.only(top: 16.0.s),
+            child: SearchSubHeader(
+              icon: Assets.svg.iconCategoriesFollowing,
+              title: context.i18n.chat_groups_subscribed,
+              count: 4,
+            ),
+          ),
+          ListView.builder(
```

Comment:
Let's extract `ListView.builder` on 33 and 68 lines into a separate widget class